README.md

```diff
@@ -25,3 +25,39 @@ sudo apt install build-essential
 ```
 go build -o rpc ./cmd
 ```
+
+### Docker image
+
+```bash
+docker build -t openamt:rpc-go .
+```
+
+and then you can run it using
+
+```bash
+$ docker run --rm -it --privileged openamt:rpc-go amtinfo
```

Comment:
In the spirit of security, let's suggest to pass only the required device that's needed. `--device /dev/mei0` should be all you need as opposed to privileged.